pkg/types/defaults/installconfig.go

```diff
@@ -80,4 +80,7 @@ func SetInstallConfigDefaults(c *types.InstallConfig) {
 	case c.Platform.None != nil:
 		nonedefaults.SetPlatformDefaults(c.Platform.None)
 	}
+	if c.Publish == "" {
```

Comment:
this seems like artifact of rebase, there's already a default being set https://github.com/openshift/installer/pull/2521/commits/8291f56ff4a475f16af7420fc7665c724d1fa703#diff-ecf2d5a25360061a409229cdf25ad9dbR50-R52

Reply:
removed.